3 reports.sql

```diff
@@ -0,0 +1,19 @@
+use RealEstateDb
+
+--1) I want a report of all the houses sorted by the block and then by the town/city
+select * from House h 
```

Comment:
Formatting tip: Questions 1 and 2 the from clause belong on the next line.

Reply:
Yeah, sorry about that.
I hope everything is better now...